modules/tinymce/src/core/main/ts/caret/LineWalker.ts

```diff
@@ -84,17 +83,23 @@ const walkUntil = (direction: VDirection, isAboveFn: PosPredicate, isBeflowFn: P
   return result;
 };
 
-const aboveLineNumber = <T extends LineClientRect>(lineNumber: number, clientRect: T) => clientRect.line > lineNumber;
-const isLineNumber = <T extends LineClientRect>(lineNumber: number, clientRect: T) => clientRect.line === lineNumber;
+const aboveLineNumber = <T extends LineClientRect>(lineNumber: number, clientRect: T): boolean =>
+  clientRect.line > lineNumber;
+
+const isLineNumber = <T extends LineClientRect>(lineNumber: number, clientRect: T): boolean =>
+  clientRect.line === lineNumber;
+
 const upUntil = Fun.curry(walkUntil, VDirection.Up, ClientRect.isAbove, ClientRect.isBelow) as (root: Node, predicateFn: RectPredicate, caretPosition: CaretPosition) => LineNodeClientRect[];
 const downUntil = Fun.curry(walkUntil, VDirection.Down, ClientRect.isBelow, ClientRect.isAbove) as (root: Node, predicateFn: RectPredicate, caretPosition: CaretPosition) => LineNodeClientRect[];
 
 const positionsUntil = (direction: VDirection, root: Node, predicateFn: RectPredicate, node: Node): LinePosClientRect[] => {
   const caretWalker = CaretWalker(root);
-  let walkFn, isBelowFn, isAboveFn,
-    caretPosition;
-  const result = [];
-  let line = 0, clientRect;
+  let walkFn;
```

Comment:
You might have forgotten one here. Or is this deliberate due to complexities in the type of the thing we're assigning this to?

Reply:
I did miss this one, thanks!